src/foam/nanos/auth/Phone.js

```diff
@@ -22,7 +22,7 @@ foam.CLASS({
               foam.nanos.auth.Phone.NUMBER,
               /^(?:\+?1[-.●]?)?\(?([0-9]{3})\)?[-.●]?([0-9]{3})[-.●]?([0-9]{4})$/);
           },
-          errorString: 'Invalid phone number.'
+          errorString: 'Please enter a valid phone number.'
```

Comment:
Oh dear god, what is our validation for phone numbers?  I would say

"Please enter a phone number in the style of 555-555-5555 or +1-555-555-5555"

Saying "Please enter a valid phone number" doesn't help anyone as we dont' tell them what makes it valid, I think give an useful example or drop the validation.

Reply:
This will have to be locale specific.  The current regex is only north america.

Reply:
Went with "Please enter a phone number". I think the regex handles lack of dashes/periods. I'm sure that adding them in after the fact if they're missing is doable but I'm not gonna do that in this PR.